index.html

```diff
@@ -55,7 +55,7 @@ <h1 class="section-title">I'm Anderson Glad to see you!</h1>
     </section>
     <section id="works-section">
```

Comment:
- [ ] I think the project images are overflowing a bit to the right. Kindly update it in accordance with the Figma template. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/206830823-98bc4db6-94c7-4d4c-84c9-bca4836fb24c.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/206830825-93313317-e273-457b-ac73-8c2f31166a8d.png" width="400">|

Reply:
Thanks for the comment. I will fixed to accomplish the template